packages/scan/src/HttpScans.ts

```diff
@@ -0,0 +1,69 @@
+import { Issue, ScanConfig, Scans, ScanState } from './Scans';
+import {
+  CreateScan,
+  GetScan,
+  ListIssues,
+  StopScan,
+  UploadFile,
+  UploadFileOptions
+} from './requests';
+import { HttpCommandDispatcher } from '@secbox/bus';
+import { inject, injectable } from 'tsyringe';
+import { CommandDispatcher } from '@secbox/core';
+
+@injectable()
+export class HttpScans implements Scans {
+  constructor(
+    @inject(CommandDispatcher)
+    private readonly commandDispatcher: HttpCommandDispatcher
```

Comment:
What is the benefit of using interfaces and IoC? 😄 https://martinfowler.com/articles/dipInTheWild.html
```suggestion
    @inject(CommandDispatcher)
    private readonly commandDispatcher: CommandDispatcher
```